components/logins/ios/Logins/RustPasswordAPI.h

```diff
@@ -28,6 +28,11 @@ Sync15PasswordEngineRaw* sync15_passwords_state_new(char const* _Nonnull db_path
                                                     char const* _Nonnull encryption_key,
                                                     Sync15PasswordsError* _Nonnull error_out);
 
+Sync15PasswordEngineRaw *sync15_passwords_state_new_encode_key_as_hex(char const* _Nonnull db_path,
```

Comment:
Nothing needs/uses this, but I added the decl here to keep it in sync with what's actually exposed by the ffi.